### ISSUE TITLE
New role idea - taken from game Feign

### ISSUE THREAD

**Issue by mogekonik1** [KARPED1EM/TownOfHostEdited#88]
![image](https://user-images.githubusercontent.com/107776037/229570215-303d20b1-2344-46ba-b26a-54b7d91db2cc.png)
Possible to do, with Detective, or Snitch

**Comment by KARPED1EM** [comment]
is it a sub-role for Detective, Snitch, psychic and something?

**Comment by mogekonik1** [comment]
> is it a sub-role for Detective, Snitch, psychic and something?

No, it's a different role, that mimics them but can't get any info.
